### DISCUSSION TITLE
A separate package for `ztracy`

### DISCUSSION THREAD

**Discussion by 7R35C0** [zig-gamedev/zig-gamedev#610] [Show and tell]
👋 Hi, I’m @7R35C0.

I have created a separate `ztracy` package, for easier import into another `build.zig.zon` file.

Here are the repos:
* [system-sdk](https://github.com/7R35C0/system-sdk) - dependency
* [ztracy](https://github.com/7R35C0/ztracy) - library
* [ztracy-tsc](https://github.com/7R35C0/ztracy-tsc) - library fix for `CPU doesn't support invariant TSC` message.
* [ztracy-example](https://github.com/7R35C0/ztracy-example) - a simple example of using library

I'm new to programming, github and all this stuff, but I hope I didn't make a mistake making these repositories and related releases.

If there is any legal or other problem with what I have done, let me know and I will act accordingly.

I kept as much of the original content as I could and declared in releases that I did not contribute to their development. I really hope there is no problem. Personally, I needed `ztracy` separately and the example it works without problems.

Thank you for your understanding and I wish you all the best!

**Comment by hazeycode** [comment]
Hi there,

I'm not a lawyer and this is not legal advice.

My understanding is that the code is MIT licensed and you free to use it however you wish so long as any license statements are retained in copies and derivations of the source. Note that more generally, libraries may have multiple licenses attached to different portions of the source which need to be observed.

Also, we are currently weighing up either moving each library to it's own repository or sticking with the current monorepo format before committing to publishing zig packages. A final decision on this will be made soon so that the libs can be easily consumed using the package manager.

Thank you for getting in touch and all the best to you!

**Comment by 7R35C0** [comment]
Hi,

I kept everything as it's in the original (your repository), including the MIT license, so personally don't see any mistakes. However, out of respect for your work, I thought it would be nice to ask for your approval.

Thank you, I wish you all the best!
